lib/schema/graphql_query.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:json_annotation/json_annotation.dart';
+
+abstract class GraphQLQuery<T, U extends JsonSerializable> {
+  GraphQLQuery({this.variables, this.query, this.operationName});
+
+  final U variables;
+  final String query;
```

Comment:
is there a way we can guarantee this and `operationName` won't be null? I thought about just instantiating `GraphQLQuery`, but I prefer this way of extending it too

Reply:
Uhm, I prefer extending too, and also given that the code is generated, it will never be generated without those overrides, so I guess it's ok to keep this way.